code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -267,6 +277,9 @@
 
 	msg += "</span>"
 
+	if(just_sleeping)
+		msg += "[p_they(TRUE)] [p_are()]n't responding to anything around [p_them()] and seems to be asleep.\n"
+
 	if(!appears_dead)
 		if(stat == UNCONSCIOUS)
 			msg += "[p_they(TRUE)] [p_are()]n't responding to anything around [p_them()] and seems to be asleep.\n"
```

Comment:
you have the same message here twice, why not rework the logic so that it does `if(stat == UNCONSCIOUS || just_sleeping)`
and then make it so that `appears_dead` is set to false if they are in fact weekend at bernies.

Reply:
Done. Thank you so much!